### PR TITLE
[15.0][MIG] procurement_purchase_no_grouping: Migration to 15.0

### PR DESCRIPTION
**Pull request by cesar-tecnativa**
@Tecnativa TT36490

### CODE REVIEW

**Comment by cesar-tecnativa** [comment]
@CarlosRoca13 please review

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by cesar-tecnativa** [comment]
@Tardo can you review?

**Comment by cesar-tecnativa** [comment]
@pedrobaeza could you please review?

**Comment by cesar-tecnativa** [comment]
@sergio-teruel could you review?

File: procurement_purchase_no_grouping/tests/test_procurement_purchase_no_grouping.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestProcurementPurchaseNoGrouping(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestProcurementPurchaseNoGrouping, cls).setUpClass()
```
Comment (pedrobaeza) [inline procurement_purchase_no_grouping/tests/test_procurement_purchase_no_grouping.py:12]:
```suggestion
        super().setUpClass()
```

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**